documentation.html

```diff
@@ -31901,16 +31940,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
-                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/smorris/ddf/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
```

Comment:
We need to fix this so the documentation does not expose the directory used by the person doing the release.